src/agiletest_cli/main.py

```diff
@@ -41,6 +42,8 @@
     default=AGILETEST_AUTH_BASE_URL,
 )
 @click.option("--timeout", help="Agiletest request timeout", default=DEFAULT_TIMEOUT)
+@click.option("--data-center", help="Agiletest Data Center Edition", default=False)
```

Comment:
should add `is_flag=True` https://click.palletsprojects.com/en/stable/options/#boolean-flags